modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -816,6 +816,23 @@ public void postProcessParameter(CodegenParameter parameter) { }
               itr.remove();
             }
         }
+
+        @SuppressWarnings("unchecked")
+        Map<String, Object> operations = (Map<String, Object>) objs.get("operations");
+        if (operations != null) {
+
+            @SuppressWarnings("unchecked")
+            List<CodegenOperation> ops = (List<CodegenOperation>) operations.get("operation");
+            for (CodegenOperation operation : ops) {
+
+                if (operation.returnType == null) {
+                    operation.returnType = "void";
```

Comment:
This will break the Java (okhttp) API client doc
```
 ### Return type
 
-null (empty response body)
+[**void**](.md)
```
We'll need to use `x-java-is-response-void` to check instead. I'll create a separate issue for track this.